### ISSUE TITLE
Fix bundling

### ISSUE THREAD

**Issue by rpearce** [rpearce/react-medium-image-zoom#199]
Looks like commonjs bundling is totally borked via rollupjs.

Issue was brought to light in https://github.com/rpearce/react-medium-image-zoom/issues/146

**Comment by rpearce** [comment]
@0x6e6562 I've found what the issue is and have patched it in a couple of other libraries. I will patch it here tonight.

**Comment by rpearce** [comment]
Should be resolved... I just tested this out in a brand new `create-react-app`:
```
$ npx create-react-app my-app
$ cd my-app
$ npm i react-medium-image-zoom@alpha
```
then
```js
import Zoom from 'react-medium-image-zoom'
import 'react-medium-image-zoom/dist/styles.css'

// ...

<Zoom>
  <img src={logo} className="App-logo" alt="logo" />
</Zoom>
```

and it looks to be working...

**Comment by 0x6e6562** [comment]
I can confirm that this issue is fixed with `^4.0.0-alpha.9`, many thanks for the fix!
